docs/drafts/developer.md

```diff
@@ -16,27 +30,7 @@ The latest version of Operator Controller can be installed with the following co
 curl -L -s https://github.com/operator-framework/operator-controller/releases/latest/download/install.sh | bash -s
 ```
 
-### Additional setup on Macintosh computers
-On Macintosh computers some additional setup is necessary to install and configure compatible tooling.
-
-#### Install Homebrew and tools
-Follow the instructions to [installing Homebrew](https://docs.brew.sh/Installation) and then execute the following to install tools:
-
-```sh
-brew install bash gnu-tar gsed
-```
-
-#### Configure your shell
-Modify your login shell's `PATH` to prefer the new tools over those in the existing environment.  This example should work either with `zsh` (in $HOME/.zshrc) or `bash` (in $HOME/.bashrc):
-
-```sh
-for bindir in `find $(brew --prefix)/opt -type d -follow -name gnubin -print`
-do
-  export PATH=$bindir:$PATH
-done
-```
-
-### Running on the cluster
+### Running on the cluster -- the targets `make run` combines
```

Comment:
I'm not sure what this heading means? Is this saying the following instructions are the individual steps contained in the `make run` command? If so, I think a more descriptive heading of that difference would be better, something like "Manual step-by-step installation" and I would make the previous 2 headings of "Installation" and "Test It Out" different as well. Maybe something like:
- Quickstart Installation
- Manual Installation

and under the Quickstart Installation you describe the 2 single step installation methods of either running make run or running the installation script
-

Reply:
done in e379ecae